lms/product/blackboard/product.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+from lms.product.blackboard._plugin.grouping import BlackboardGroupingPlugin
+from lms.product.product import Plugins, Product
+
+
+@dataclass
```

Comment:
Not a fan of data classes here, not holding that much data.

In any case if we regret the decision it not a big change.

Reply:
It's just because it means we don't have to write a lot of methods, and the types will be more strongly enforced by linting etc.